### DISCUSSION TITLE
How to run dev version of Directus v9 on Windows 10?

### DISCUSSION THREAD

**Discussion by dergachevm** [directus/directus#5115] [Archive]
Hey there. I am trying to run dev version of Directus with this guide
https://docs.directus.io/contributing/running-locally/
and I don't see any chance to do it.
Through WSL2 (with powershell same problems) it always warn about missing packages.
https://prnt.sc/11jgdh7
`npm run cli bootstrap` - same result https://prnt.sc/11jgfwh

After that I am trying to do `npm install` manually for each folder: api, app, docs, packages (in each folder too) but problem still there.

What I am missing?

**Comment by rijkvanzanten** [comment]
Are you using Node 15 / NPM 7+?

**Reply by dergachevm** [reply]
Damn, no, v14.15.0
With 15.14.0 all done well. That's first string in manual - You will need to have the latest version of Node. 😟
Thanks!
